src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,39 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
+		$section_classes = [ 'tribe-events-widget-admin-form__input-section' ];
+		if ( ! empty( $field['classes'] ) ) {
```

Comment:
Smaller optimization:
```suggestion
		if ( ! empty( $field['classes'] ) ) {
			$section_classes = array_merge( $section_classes, Arr::list_to_array( $field['classes'], ' ' ) );
		}

```

Reply:
This kind of small optimization could be done everywhere this kind of logic is applied.